examples/index.json

```diff
@@ -2318,6 +2318,15 @@
                 "disableVerificationReason": "Has different naming convention",
                 "disablePlayground": true,
                 "isLearnByExample": false
+            },
+            {
+                "name": "Set deadline",
```

Comment:
```suggestion
                "name": "Check deadline",
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-distribution/pull/3544/commits/d1bc7441bf9e7517b363d6378995f0ae11ef6ab7